curated_transformers/tokenization/auto_tokenizer.py

```diff
@@ -0,0 +1,121 @@
+from typing import Dict, Optional, Type, cast
+
+from huggingface_hub import HfFileMetadata, get_hf_file_metadata, hf_hub_url
+from huggingface_hub.utils import EntryNotFoundError
+
+from ..util.hf import get_hf_config_model_type
+from .bert_tokenizer import BertTokenizer
+from .camembert_tokenizer import CamembertTokenizer
+from .hf_hub import FromHFHub, get_tokenizer_config
+from .llama_tokenizer import LLaMATokenizer
+from .roberta_tokenizer import RobertaTokenizer
+from .tokenizer import Tokenizer, TokenizerBase
+from .xlmr_tokenizer import XlmrTokenizer
+
+TOKENIZER_JSON = "tokenizer.json"
```

Comment:
Could we move this and the private methods below to `util.hf`?

Reply:
I moved `get_file_metadata` and this constant, but I don't think `get_tokenizer_class_from_config` belongs in `hf_util`, since it's not a reusable function and specific to `AutoTokenizer`.